docs/freqtrade-101.md

```diff
@@ -0,0 +1,106 @@
+# Freqtrade Strategies 101: A Quick Start for Strategy Development
+
+For the purposes of this quick start, we are assuming you are familiar with the basics of trading.
+
+## Required Knowledge
+
+A strategy in Freqtrade is a Python class that defines the logic for buying and selling cryptocurrency `assets`.
+
+Assets are defined as `pairs`, which represent the `coin` and the `stake`. The coin is the asset you are trading using another currency as the stake.
+
+Data is supplied by the exchange in the form of `candles`, which are made up of a six values: `date`, `open`, `high`, `low`, `close` and `volume`.
+
+`Technical analysis` functions analyse the candle data using various computational and statistical formulae, and produce secondary values called `indicators`.
+
+Indicators are analysed on the asset pair candles to generate `signals`.
+
+Signals are turned into `orders` on a cryptocurrency `exchange`, i.e. `trades`.
+
+We use the terms `entry` and `exit` instead of `buying` and `selling` because Freqtrade supports both `long` and `short` trades.
+
+- long:
+  - You buy the coin based on a stake, e.g. buying the coin BTC using USDT as your stake, and you make a profit by selling the coin at a higher rate than you paid for. Profits are made in long trades by the coin value going up versus the stake.
+- short
+  - You borrow capital from from the exchange in the form of the coin, and you pay back the stake value of the coin later. Profits are made in short trades by the coin value going down versus the stake (you pay the loan off at a lower rate).
```

Comment:
I'm not sure i like the nested list here (from a rendering perspective) - it kinda breaks reading flow 
maybe either write long/short bold (instead of as list) - or have 1 level of lists (something like the below?)


```suggestion
- **long**: You buy the coin based on a stake, e.g. buying the coin BTC using USDT as your stake, and you make a profit by selling the coin at a higher rate than you paid for. Profits are made in long trades by the coin value going up versus the stake.
- **short**: You borrow capital from from the exchange in the form of the coin, and you pay back the stake value of the coin later. Profits are made in short trades by the coin value going down versus the stake (you pay the loan off at a lower rate).
```